phovea_security_flask/flask_login_impl.py

```diff
@@ -144,7 +151,7 @@ def _load_user_from_key(self, api_key):
 
   def _load_user_from_request(self, request):
     # first, try to login using the api_key url arg
-    api_key = request.args.get('api_key')
+    api_key = request.headers.get('api_key')
```

Comment:
See my question in https://github.com/phovea/phovea_security_flask/pull/27#issuecomment-524932109